src/Compilers/Core/Portable/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -277,6 +277,7 @@ public static TRoot ReplaceTrivia<TRoot>(this TRoot root, SyntaxTrivia trivia, S
         }
 
         internal const string DefaultIndentation = "    ";
+        internal const string DefaultEOL = "\r\n";
```

Comment:
It is a bit annoying that the API's default line-ending-sequence on Linux is the Windows sequence.